Sources/PointFreeTestSupport/MockEpisode.swift

```diff
@@ -20,9 +20,9 @@ private let proofInFunctions = Episode(
   transcriptBlocks: [
     Episode.TranscriptBlock(
       content: """
-      Swift’s generic functions allow us to explore a beautiful idea that straddles the line between \
-      mathematics and computer science. If you write down and implement a function using only generic data \
-      types, there is a corresponding mathematical theorem that you have proven true. There are a lot of \
+      Swift’s generic `functions` allow us to **explore** a beautiful idea that straddles the line between \
+      __mathematics__ and computer science. If you write down and implement a function using only generic data \
```

Comment:
I'm gonna redo the mock episode to be a better example of all the weird edge cases that can show up in a transcript